docs/sources/clients/promtail/gcplog-cloud.md

```diff
@@ -92,6 +102,36 @@ $ gcloud pubsub subscriptions create cloud-logs --topic=projects/my-project/topi
 
 For more fine grained options, refer to the `gcloud pubsub subscriptions --help`
 
+### Push
+
+Since GCP PubSub push subscriptions is a rather new service, one needs to grant Google a permission in some cases. First, check the date the GCP project was created on:
+
+```bash
+> gcloud projects describe $GCP_PROJECT_ID
+
+createTime: 'some date'
+...
+projectId: $GCP_PROJECT_ID
+projectNumber: '$GCP_PROJECT_NUMBER'
+```
+
+If the `createTime` is later than **April 8, 2021**, skip the following step. Otherwise, you need to grant the `iam.serviceAccountTokenCreator` role to a Google-managed service account:
```

Comment:
Nice details. can we link any official gcloud docs as the reference here? just for the authenticity of the **April 8, 2021** part.

Reply:
Yes! I'll link the section that contains it, even though it's preety hidden

Reply:
Added